src/plugins/data/public/search/expressions/es_raw_response.test.ts

```diff
@@ -0,0 +1,827 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { EsRawResponse, esRawResponse } from './es_raw_response';
+
+jest.mock('@kbn/i18n', () => {
+  return {
+    i18n: {
+      translate: (id: string, { defaultMessage }: { defaultMessage: string }) => defaultMessage,
+    },
+  };
+});
+
+describe('esRawResponse', () => {
+  describe('converts aggregations to table', () => {
```

Comment:
Could you add a test for a multi-level aggregation, for example Terms + Date histogram, or Terms + Terms? I see that it's working the way I expected, but it wasn't clear until I actually ran this locally.

<img width="956" alt="Screen Shot 2020-08-13 at 5 44 27 PM" src="https://user-images.githubusercontent.com/666475/90190301-bf211e00-dd8c-11ea-8717-0bb83af6366b.png">